### PR TITLE
feat: supported application automation from ui

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

Add automation application

https://github.com/open-metadata/OpenMetadata/assets/33024356/44400a7c-8635-4717-8bf6-67e74973417d

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by harshach — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/Applications/AppInstall/AppInstall.component.tsx [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+  if (!appData) {
+    return (
+      <ErrorPlaceHolder>
+        {getEntityMissingError('application', fqn)}
+      </ErrorPlaceHolder>
+    );
+  }
+
+  return (
+    <PageLayoutV1 pageTitle={t('label.application-plural')}>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Applications/AppInstall/AppInstall.component.tsx:186]: I think it's a Page and should be part of Page directory instead components

File: openmetadata-ui/src/main/resources/ui/src/components/Applications/AppInstall/AppInstall.component.tsx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+      <Row>
+        <Col span={24}>
+          <IngestionStepper
+            activeStep={activeServiceStep}
+            steps={STEPS_FOR_APP_INSTALL}
+          />
+        </Col>
+        <Col span={24}>
+          <div className="p-md flex-center">{RenderSelectedTab()}</div>
+        </Col>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Applications/AppInstall/AppInstall.component.tsx:196]: Can we render this as `Tabs` component

File: openmetadata-ui/src/main/resources/ui/src/pages/Application/ApplicationPage.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    history.push(ROUTES.MARKETPLACE);
+  };
+
+  useEffect(() => {
+    fetchApplicationList();
+  }, []);
+
+  if (isLoading) {
+    return <Loader />;
+  }
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/Application/ApplicationPage.tsx:93]: No data placeholder is missing, Not sure if required 
For Card layout we can use Card based skeleton instead Loader
Reder Persona listing page

File: openmetadata-ui/src/main/resources/ui/src/pages/MarketPlacePage/MarketPlacePage.tsx [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    currentPage,
+    paging,
+    pageSize,
+    handlePagingChange,
+    handlePageChange,
+    handlePageSizeChange,
+  } = usePaging();
+  const history = useHistory();
+  const [isLoading, setIsLoading] = useState(true);
+  const [applicationData, setApplicationData] =
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/MarketPlacePage/MarketPlacePage.tsx:49]: What's difference between this an Applications page?

File: openmetadata-ui/src/main/resources/ui/src/styles/app.less [hunk 5]
```diff
@@ -738,3 +738,48 @@ a[href].link-text-grey,
...
+
+.lazy-log-container {
+  .react-lazylog-searchbar {
+    position: absolute;
+    z-index: 9;
+    top: -45px;
+    background: unset;
+    padding: unset;
+
+    .react-lazylog-searchbar-input {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/styles/app.less:752]: I won't prefer it on app.less
Advised to use specific component less

**Review by chirag-madlani — APPROVED** [review]
@karanh37 @Ashish8689 
Left minor comments which can be addressed on next PR
Nothing related to feature  🚀

**Pull request merged**